finality-aleph/src/abft/traits.rs

```diff
@@ -4,6 +4,7 @@ use std::{cmp::Ordering, fmt::Debug, hash::Hash as StdHash, marker::PhantomData,
 
 use codec::{Codec, Decode, Encode};
 use futures::{channel::oneshot, Future, TryFutureExt};
+use network_clique::SpawnHandleT;
```

Comment:
Not sure I explained myself well. I do not mean to change `impl SpawnHandleT` to `impl network_clique::SpawnHandleT` in this file. 

I mean to:
* Keep `SpawnHandleT` trait definition that is removed from `finality-aleph` in this PR
* Impl `SpawnHandleT for SpawnHandle` and use it throughout the `finality-aleph` as `crate::SpawnHandleT` instead of `network_clique::SpawnHandleT`
* do `impl network_clique::SpawnHandleT for SpawnHandle {` and use it for clique network service

I think it is a little bit confusing to `use network_clique::SpawnHandleT` in places in code that are not related to clique network.

Reply:
I agree that this is a bit weird, but I think it's fine for now. There are at least some other related problems (e.g. `crypto` being in `network_clique`), so maybe they should be solved all together in a separate task. Especially since I'm not even sure what the proper, full solution would be.